shared/chat/conversation/rekey/container.js

```diff
@@ -0,0 +1,69 @@
+// @flow
```

Comment:
just make our own connector so the conversation container doesn't need to understand these things